backend/src/controllers/health-api-controller.ts

```diff
@@ -1,9 +1,12 @@
-// Simple health endpoint to check API status
+/**
+ * @desc Simple health endpoint to check API status
+ * @author GDamaso
+ */
 import { Response, Request } from 'express';
 
-// If we're healthy...
+
 const checkHealth = (req: Request, res: Response) => {
-  res.status(200).send('NMP API is healthy and ready!');
+  res.status(200).send('NMP API is healthy and ready! Hot reloading all the way!');
```

Comment:
```suggestion
  res.status(200).send('NMP API is healthy and ready!');
```
This response is used for your health check